vecnet/vecnet_linux_386.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vecnet
+
+import (
+	"syscall"
+)
+
+//TODO / FIXME: pkg syscall doesn't seem to define constants needed to make the Recvmsg syscall directly (through .Syscall())
```

Comment:
do you need this file? it should compile and just not use the optimized version if you remove it, i thiiink?

Reply:
Good call. Looks like There was some more that could be reduced as well.
https://github.com/hugelgupf/p9/pull/12/commits/3ac24bae1de6a71fdee2a9be1a2c0f77b7782caa